internal/types/instance.go

```diff
@@ -142,6 +142,10 @@ func (r *InstancesRequest) Validate() error {
 			return fmt.Errorf("invalid hostname at index %d: %w", i, err)
 		}
 
+		if len(instance.Volumes) == 0 {
```

Comment:
This entire for loop should just be a single call to `InstanceRequest.Validate`.

Reply:
https://github.com/celestiaorg/talis/issues/197